screen/settings_menu.h

```diff
@@ -13,5 +13,5 @@ class settings_menu : public screen {
     std::vector<button*> buttons;
     std::vector<slider*> sliders;
 
-    virtual View_mode Run(sf::RenderWindow& window) override;
+    virtual View_mode Run(sf::RenderWindow& window, std::shared_ptr<Player> player) override;
 };
```

Comment:
`std::shared_ptr<Player> player` also add nullptr as a default to clarify code function